integreat_cms/cms/templates/authentication/register_totp.html

```diff
@@ -0,0 +1,32 @@
+{% extends "authentication/_base.html" %}
+{% load i18n %}
+{% block heading %}
+    <i icon-name="smartphone"></i>
+    {% trans "Add authenticator app" %}
+{% endblock heading %}
+{% block content %}
+    <p class="mt-4">
+        <i icon-name="camera" class="pb-1"></i> {% trans "Please scan the QR code with your authenticator app:" %}
+    </p>
+    <img width="100%"
+         height="100%"
+         alt="QR code for TOTP registration"
+         src="{{ qr }}"/>
+    <p class="mb-3">
+        {% blocktrans trimmed with BRANDING=BRANDING|capfirst %}
+            Please enter the 6-digit code for "{{ BRANDING }}" to  confirm the registration:
+        {% endblocktrans %}
+    </p>
+    <form method="post"
+          action="{% url 'register_totp' %}"
+          class="flex flex-nowrap gap-4">
```

Comment:
```suggestion
    <form method="post" autocomplete="off" class="flex flex-nowrap gap-4">
```
This url is only accessible for staff users, so I guess it's better to just POST to the same url which is currently opened (which should already have the region slug for region users).
Also, I guess disabling autocomplete probably makes sense for OTPs...